src/pages/messages/TasksPage.tsx

```diff
@@ -225,13 +225,15 @@ const TasksPage = () => {
         }>
         <SideNavHeader icon={AssignmentIcon} text={t('common.tasks')} />
 
-        <OrganizationScope
-          organizationScope={organizationScope}
-          setOrganizationScope={setOrganizationScope}
-          excludeSubunits={excludeSubunits}
-          setExcludeSubunits={setExcludeSubunits}
-          hide={isOnCorrectionListPage}
-        />
+        {!isOnTicketsPage && (
```

Comment:
good spot :D kommer sikkert oppgave for ågjøre samme jobben på NVI også etter hvert